### DISCUSSION TITLE
azura cast does not upload metadata streamers

### DISCUSSION THREAD

**Discussion by Burchelwnd** [AzuraCast/AzuraCast#4918] [Q&A]
Hello
I installed azuracast. I installed shoutcast.
unfortunately, but after connecting via the azura streamer, it does not send data to the server, only the name of the radio is permanently in the name of the broadcast and the streamer misc. with radioboss and butt.
I am asking for help because I did not find the answer here on the forum.

**Comment by BusterNeece** [comment]
@Burchelwnd Hello! Please follow this issue on the Liquidsoap GitHub for more information on this known issue: savonet/liquidsoap#2109

**Reply by Burchelwnd** [reply]
unfortunately, there is no information in this post

**Comment by Burchelwnd** [comment]
Ok thanks.
